git-link.el

```diff
@@ -496,6 +513,11 @@ return (FILENAME . REVISION) otherwise nil."
       (git-link--read-remote)
     (git-link--remote)))
 
+(defun empty-string-p (string)
+  "Return true if the STRING is empty or nil. Expects string type."
+  (or (null string)
+      (zerop (length (string-trim string)))))
```

Comment:
Is there a need to trim? If not maybe use `string-empty-p`? I had tried to avoid for compatability but at this point maybe safe to require minimum of 24.4?

Reply:
DONE